### ISSUE TITLE
Inexact information regarding LDAP querying DN - does not have to be an admin at all with openLDAP

### ISSUE THREAD

**Issue by git-noise** [PowerDNS-Admin/PowerDNS-Admin#913]
Hello,

The following webpage:
     admin/setting/authentication#tabs-ldap

Displays:
``
  ADMINISTRATOR INFO
      Your LDAP connection string and admin credential used by PDA to query user information.
``
That is not necessarily true , as in default config for openLDAP, any user that can query the directory information - even anonymous_bind - may be enough.
If anonymous bind is disabled - not by default in openLDAP latest versions - any authenticated user usually can:
1- Query info in its directory
2- Use userPassword with the 'auth' privilege, which allows for authentication without having read access.

To summarize, any authenticated user - and sometimes even anonymous bind - may have enough rights to allow LDAP config to work.

Best,
